src/System.Private.ServiceModel/src/windows/project.json

```diff
@@ -3,11 +3,11 @@
   "frameworks": {
     "netcore50": {
```

Comment:
Don't need to change now. I think we can remove netcore50 section in the future since we are harvesting netcore50 now.